AmplifyPlugins/Core/AWSPluginsCore/Model/Support/AuthRule+Extension.swift

```diff
@@ -0,0 +1,18 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Amplify
+
+extension AuthRule {
+    func getOwnerFieldOrDefault() -> String {
+        return (ownerField != nil) ? ownerField!.stringValue : "owner"
```

Comment:
I think it would be better to use guard/let over force unwrap
```
        guard let ownerField = ownerField else {
            return "owner"
        }
        return ownerField.stringValue
```